File: src/main/java/assertion/pipe/PipeAssert.java
package assertion.pipe;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class PipeAssert {
    public static void pipeAssertion(String pipeResponse, Integer number, String assertingValue) {
        String[] splittedPipeResponse = pipeResponse.split("\\|");
        assertEquals(assertingValue, splittedPipeResponse[number]);
    }
}

File: src/main/java/assertion/pipe/PipeAssert.kt
package assertion.pipe

import org.junit.jupiter.api.Assertions

object PipeAssert {
    fun pipeAssertion(pipeResponse: String, number: Int, assertingValue: String) {
        val splittedPipeResponse = pipeResponse.split("\\|".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        Assertions.assertEquals(assertingValue, splittedPipeResponse[number])
    }
}
